janus_core/calculations/md.py

```diff
@@ -607,7 +618,12 @@ def _run_dynamics(self) -> None:
             if self.logger:
                 self.logger.info("Beginning temperature ramp at %sK", temps[0])
             for temp in temps:
+                if isclose(temp, 0.0):
+                    continue
```

Comment:
Similar to @oerc0122's comment, if we were to decrease the temperature to 0, wouldn't this mean the velocity distribution was wrong here?

Reply:
ok I did not understand the comment let me think about it... why shall it be wrong?

Reply:
I was thinking for the same reason as why `set_velocity_distribution` is necessary in general.

An extreme example, but if `temps = [100, 50, 0]`, then this would call `_set_velocity_distribution` at T=100 and T=50, but not set it for T=0, so the velocity distribution would be wrong (non-zero) for the final `dyn.run`?

Possibly the fact that we still set `temperature_K=0` means this isn't a problem though?

Reply:
ok just checked it... as long as step is negative shall be fine, updated to account for temp_end being possible 0